components/Cards/SchoolCard/__tests__/SchoolCard.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import SchoolCard from '../SchoolCard';
+
+const locations = [
+  {
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    state: 'CA',
+    va_accepted: true,
+    zip: 94111,
+  },
+  {
+    address1: '123Battery Street',
+    address2: 'Ste 1',
+    city: 'San Diego',
+    state: 'CA',
+    va_accepted: false,
+    zip: 90111,
+  },
+];
+
+describe('SchoolCard', () => {
+  let componentInstance;
+  let wrapper;
+  let onClickMock;
+  beforeEach(() => {
+    componentInstance = (
+      <SchoolCard
+        cardFlipCallback={onClickMock}
+        hasHardwareIncluded
+        hasHousing
+        hasOnline
+        hasOnlyOnline={false}
+        isFullTime
+        locations={locations}
+        logoSource="source"
+        name="school name"
+        website="website"
+      />
+    );
+    onClickMock = jest.fn();
+    wrapper = mount(componentInstance);
+  });
+
+  it('should render with required props', () => {
+    createShallowSnapshotTest(componentInstance);
+  });
+
+  it('should change state when showBackOfCard/showFrontOfCard function is called', () => {
```

Comment:
You can delete this test now - you've added a better test covering the same thing on line 61.